gammapy/estimators/utils.py

```diff
@@ -226,3 +228,54 @@ def resample_energy_edges(dataset, conditions={}):
                 energy_edges.append(energy_min)
                 break
     return u.Quantity(energy_edges[::-1])
+
+
+def compute_lightcurve_fvar(lightcurve, quantity="flux"):
+    r"""Compute the fractional excess variance of the input lightcurve.
+
+    Internally calls the `~gammapy.stats.compute_fvar` function
+
+
+    Parameters
+    ----------
+    lightcurve : '~gammapy.estimators.FluxPoints'
+        the lightcurve object
+    quantity : `string`
+        specific quantity contained in the FluxPoints object used for calculation.
+        Useful in case of incomplete lightcurves computed outside gammapy
+
+    Returns
+    -------
+    fvar : `~numpy.ndarray`
+        Array of fractional excess variance and associated error for each energy bin of the lightcurve.
+    """
+
```

Comment:
I think `data = getattr(lightcurve, ...)` might be useful here. You can check the supported quantities beforehand...

Reply:
Thank you very much Axel! The new committed version implements your suggestion to check the supported quantities.